### PR TITLE
Add a sample function

### PR DESCRIPTION
**Pull request by cgwy**
blah blah

Signed-off-by: KAi <kkw@google.com>

### CODE REVIEW

File: class composition [hunk 1]
```diff
@@ -5,6 +5,11 @@ class Bookshelf:
     
     def __str__(self) :
         return f'Bookshelf with {len(self.books)} books.'
+        
+    def __repr__(self):
+        # just a random sample
```
Comment (cgwy) [inline class composition:10]: Why add a sample?

File: class composition [hunk 2]
```diff
@@ -5,6 +5,11 @@ class Bookshelf:
...
+        # TODO(tutu): delete this later.
+        return f'Bookshelf with {len(self.books)} books.'
```
Comment (cgwy) [inline class composition:12]: Why is it called Bookshelf?

File: class composition [hunk 3]
```diff
@@ -5,6 +5,11 @@ class Bookshelf:
     
     def __str__(self) :
```
Comment (cgwy) [inline class composition:6]: Why underscore

File: class composition (hunk 2 above)
Comment (saratutu) [reply class composition:12]: emmm

**Pull request closed without merging**